uwblueprint/community-fridge-kw
# feat: Update my account page for volunteers

## Brief description. What is this change? 
Updated My Account page so that UI is changed for volunteers and volunteers can update their info


### [Update my account page for volunteers](https://www.notion.so/uwblueprintexecs/Team-Hub-Phase-2-cf7744a5b04748059761c1649a61a463?p=c6710305429045169b7ab8315fb82143)



## Implementation description. How did you make this change?
* UI changes
* Only update donor info if user's role is donor


## Steps to test
1. Create an account/log in as a volunteer.
2. Go to My Account page and edit first name, last name, or phone number.
3. When you refresh the page, or log out and log in, the fields should be updated.


## Checklist
